README.md

```diff
@@ -150,7 +150,12 @@ The following output formats are supported:
 
 ## Configuration
 
-The configuration is done via arguments or an `.ecrc` file.
+The configuration is done via arguments or it will take the first config file found with the following file names:
+
+- `.ecrc`
+- `.ecrc.json`
+- `.editorconfig-checkerrc`
+- `.editorconfig-checkerrc.json`
```

Comment:
I'd like to keep the `rc` from the filename, because it is a known convention in Unix/Linux, and many other tools also follow this convention.

Reply:
There is no use for so much format

Reply:
The rc suffix seems odd to me once you add an extension.

Also, you are adding too much formats, these one should be enough

- `.ecrc`
- `.editorconfig-checker.json`

Here is what I would suggest, and I don't think there is a need for the second one.

I could accept supporting

- `.ecrc`
- `.editorconfig-checkerrc.json`

But then this is it.

Then about the .json5 extension, klaernie suggested using .json5 extension

Reply:
I sadly know a couple dyslexic colleague who will fail to notice the doubled letter r in `checkerrc`. While your of cause correct that many tools use that convention, they equally support the convention that binary names should be as short as possible - trading readability for brevity. And this is exactly the pattern we're currently fixing here. Plus I just had to actually Google what `rc` stands for - and sadly "runtime configuration" is not what I found.

In my mind `.editorconfig-checker.json` is readable and understandable. But that is of cause only my opinion, and I'm not willing to die on a hill for it ;)

Reply:
> Then about the .json5 extension, klaernie suggested using .json5 extension

I did, but am becoming a bit wary of it - it requires us to replace a parser provided by the standard library with one only found in external modules, forcing us to make a choice between the implementations. Also some voices on the Internet argue against JSON5 and for other competing standards trying to achieve the same goals. So I'm internally divided on what the best way is.

Reply:
Okay, yeah, good point.
>  trading readability for brevity. And this is exactly the pattern we're currently fixing here.

`.editorconfig-checker.json` should be good. :smile: